doc/topics/jinja/index.rst

```diff
@@ -1148,6 +1148,40 @@ Returns:
     }'
   }
 
+
+.. jinja_ref:: traverse
+
+``traverse``
+--------------
```

Comment:
The doc checks might trigger a warning here (the line of `-` must be as long as the test above).

Reply:
Ok